api.go

```diff
@@ -2901,14 +2901,24 @@ func (sc *Client) DeleteEdgeDeployment(corpName, siteName string) error {
 	return err
 }
 
+type CreateOrUpdateEdgeDeploymentServiceBody struct {
+	ActivateVersion      bool          `json:"activateVersion,omitempty"`      //Activate Fastly VCL service after clone
+	PercentEnabled       int           `json:"percentEnabled,omitempty"`       //Percentage of traffic to send to NGWAF@Edge
```

Comment:
I think these parameters need to be pointers for them to be omitted and optional.

Reply:
Might be able to get away without making PercentEnabled a pointer, since its zero value is the default, and omitempty will cause it to not be generated if it is zero.

But ActivateVersion should be a pointer, since the default should be true, not the zero value.

Reply:
@mattg-sigsci @dkegel-fastly I've changed `ActivateVersion` to a pointer [here](https://github.com/signalsciences/go-sigsci/pull/51).